requirements.txt

```diff
@@ -1,7 +1,4 @@
 numpy
 jaxlib
 jax
-# tensorflow is needed for tensorflow-datasets
```

Comment:
If we remove this from Flax core requirements, I think we need to add them to a new requirements.txt and setup.py files (or maybe just one of them?) inside the examples/ directories

Reply:
I just found that we are also using it in `checkpoint.py`. I will tweak this a bit more.
I'll add requirements.txt to flax/examples